Vagrant_Recommendation/Vagrantfile

```diff
@@ -0,0 +1,70 @@
+# -*- mode: ruby -*-
```

Comment:
we only need this Vagrantfile, please do not put it under another directory. You can follow https://github.com/nyu-devops/lab-flask-rest to see the architecture or the project.

Reply:
I have already update it, thanks for your remind.